apps/api/.env.sample

```
@@ -57,3 +57,12 @@ SUPER_ADMIN_ACCOUNT_EMAIL=
 TRELLO_KEY=
 
 FFMPEG_VERSION=5.0.1
+
+# GAUZY CONFIG
+GAUZY_API_URL=https://apistage.gauzy.co/api
+
+# JWT SECRET
+REC_JWT_SECRET=hard!To_find:my/GuyZ
+REC_JWT_EXPIRES_IN=1h
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The REC_JWT_EXPIRES_IN key should go before the REC_JWT_SECRET key

(UnorderedKey)